ui/admin/app/templates/scopes/scope/host-catalogs/new.hbs

```diff
@@ -19,6 +19,7 @@
       @submit={{route-action 'save' @model}}
       @cancel={{route-action 'cancel' @model}}
       @changeType={{route-action 'changeType'}}
+      @doc='host-catalog-set-up-cloud-hosts'
```

Comment:
I think this should be: `host-catalog.set-up-cloud-hosts` ?

Reply:
oops, you are right. that's the pattern we follow